docs/user_guide/architecture.md

```diff
@@ -796,6 +796,14 @@ scheduler will:
    response to the inference request with the tensors mapped to the ensemble
    output names.
 
+Unlike other models, ensemble models do not support "instance_group" field in
+the model configuration. The reason is that the ensemble scheduler itself
+is mainly an event-driven scheduler with very minimal overhead so its
+almost never the bottleneck of the pipeline. For optimized throughput,
+make sure to optimize your pipeline using
+[Model Analyzer](https://github.com/triton-inference-server/model_analyzer)
+for best performance.
```

Comment:
Re-wording suggestion
```suggestion
almost never the bottleneck of the pipeline. The composing models
within the ensemble can be individually scaled up or down with their
respective `instance_group` settings. To optimize your model pipeline
performance, you can use
[Model Analyzer](https://github.com/triton-inference-server/model_analyzer)
to find the optimal model configurations.
```

Reply:
thanks for the suggestion. Fixed.

Reply:
I'm not seeing the update on my end, do you?

Reply:
Oops, looks like push failed. Should be available now.